render/webgl/src/lib.rs

```diff
@@ -988,7 +1005,9 @@ impl RenderBackend for WebGlRenderBackend {
     }
 
     fn unregister_bitmap(&mut self, bitmap: BitmapHandle) {
-        self.bitmap_registry.remove(&bitmap);
+        if let Some(entry) = self.bitmap_registry.remove(&bitmap) {
```

Comment:
I think we can `impl Drop for RegistryData`, and move `delete_texture` to there.
This is both more clear and less error-prone.